seminars/sem3/hw3/task1.py

```diff
@@ -0,0 +1,27 @@
+"""
+16: Требуется вычислить, сколько раз встречается некоторое число X в списке A. 
+Пользователь в первой строке вводит натуральное число N – количество элементов в списке. 
+В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). 
+Последняя строка содержит число X.
+
+5
+    7 -2 3 5 1
+    3
+    -> 1
+"""
+from random import randint
+
+size = int(input('Введите размер списка: '))
+lst = [randint(-10, 10) for _ in range(size)]
+print(lst)
+
+num = int(input('Введите число, которое хотите найти: ')) # вывожу запрос в новой строке, т.к. пользователю нужно видеть распечатанный список
+
+count = 0
+closest = lst[0]
+
+for i in lst:
+    if i == num: count += 1
+    if abs(num - i) < abs(num - closest): closest = i
```

Comment:
Такая запись хочет допустима, но не рекомендуется.
```suggestion
    if i == num:
        count += 1
    if abs(num - i) < abs(num - closest):
        closest = i
```